keyboards/hfdkb/keyboard_sw/mk47/matrix.c

```diff
@@ -0,0 +1,199 @@
+/* Copyright (C) 2022 jonylee@hfd
```

Comment:
All of this should be moved to `mk47.c` file.

Reply:
ok